src/domains/Invite/types.ts

```diff
@@ -0,0 +1,19 @@
+export interface InviteForm {
+	MoC: string | undefined,
+	topic: string | undefined,
+	eventDateTime: string | undefined,
+	constituentScope: 'state' | 'district',
+	region: string | undefined,
+	deliveryTime: Date | undefined,
+	townHallId: string | undefined,
+}
```

Comment:
Hmm, what's the purpose of declaring these as `string | undefined` as opposed to `MoC?: string`.